src/components/scenes/FioSentRequestDetailsScene.js

```diff
@@ -34,7 +35,7 @@ class FioSentRequestDetailsComponent extends React.PureComponent<Props> {
     const fiatPerCrypto = exchangeRates[rateKey] ? exchangeRates[rateKey] : 0
     const amountToMultiply = parseFloat(amount)
```

Comment:
amount is already a string you can use it directly in bns.mul() without converting to and from a number.